common/src/services/search.service.ts

```diff
@@ -51,7 +51,10 @@ export class SearchService implements SearchServiceAbstraction {
     const builder = new lunr.Builder();
     builder.ref("id");
     builder.field("shortid", { boost: 100, extractor: (c: CipherView) => c.id.substr(0, 8) });
-    builder.field("name", { boost: 10 });
+    builder.field("name", {
+      boost: 10,
+      extractor: (c: CipherView) => c.name.normalize("NFD").replace(/[\u0300-\u036f]/g, ""),
```

Comment:
Shouldn't we add the same extractor for most user-input fields?
* subtitle
* username
* notes
?

Reply:
Do we also need to also strip out the characters from the search input as well? With this change searching for `náme` would no longer match `nàme` since it would be normalized as `name`.

Reply:
@MGibson1 yeah I would add them to those fields. @Hinton the accented characters are currently being striped off from the input field.